code/modules/mob/living/carbon/human/life.dm

```diff
@@ -331,13 +334,187 @@
 
 		oxygen_alert = max(oxygen_alert, 1)
 		return 0
-	var/obj/item/organ/internal/lungs/L = internal_organs_by_name[BP_LUNGS]
-	if(L && L.handle_breath(breath))
+
+	if(get_organ_efficiency(OP_LUNGS) && handle_breath_lungs(breath))
```

Comment:
You can simplify this code in the following way:: 
if(get_organ_efficiency(OP_LUNGS))
   failed_last_breath = handle_breath_lungs(breath)

Reply:
Fixed.